123.js

```diff
@@ -1,4 +1,4 @@
-var a=10;
+var a=100;
 var b=20;
 var c=30;
```

Comment:
@Harry-Zhong Please check it out.
```suggestion
var b=200;
var c=300;
```